### DISCUSSION TITLE
Give us access to Outland and Northrend glyphs through the Roboto NPC.

### DISCUSSION THREAD

**Discussion by minthes** [chromiecraft/chromiecraft#3501] [Suggestions]
Some classes/specs are struggling right now because they are missing key glyphs. For example, while paladins have access to [Glyph of Righteous Defense], warriors do not have access to [Glyph Of Taunt]. This makes warrior itemization awkward.

You could make these glyphs accessible by allowing players to turn in high level herbs/stacks of high level herbs in exchange for them through the Roboto NPC.

**Comment by Never6** [comment]
I completly agree
Gimme barskin glyph

**Comment by ghost** [comment]
Stoneclaw glyph for shamans please

**Comment by Si1ker** [comment]
+1 on this, please.

~ sílker

**Comment by heiligerhuegel** [comment]
++ yes!

**Comment by Soulex23** [comment]
At the very least it'll help expand the flavor of some classes, though I do need to look into Inscription as a profession beyond vanilla content

**Comment by plasma-93** [comment]
Great idea

**Comment by BeefSteaker** [comment]
They should be made available to inscriptors.

**Comment by Chobungus** [comment]
+1 8) good idea

**Comment by Dupeloop** [comment]
No this is a slow progression server we will get the glyphs when the come out. Otherwise give us the max enchants too. If ur impatient go to a server like warmane that has faster progression. Stop rushing content on this server. If it takes is 2 years to get to TBC glyphs then so be it.

**Reply by BeefSteaker** [reply]
Enchants are intentionally gated. Glyphs werent introduced until wotlk and aren't supposed to be restricted

**Comment by Vitusin** [comment]
Any News?

**Comment by balleny** [comment]
exactly the point. Glyphs were released with Wrath of the Lichking, making all glyphs accessible directly (technically, you needed to discover them, but plenty of people picked up the profession, thus everything was available day 1).
Restricting access to the glyphs now, causes classes to be further limited / restricted, potentially even imbalanced.

**Comment by Hacki95** [comment]
We currently have no plans to add the remaining glyphs before their respective content becomes available.

There are several reasons why we don't want to introduce them right now:

- This would mean further changes to the class balance. Once its out we can't take it back, possibly resulting in a lot of work for us trying to rebalance content/pvp around the new glyphs.
- It would take away more content from the later brackets.
- Its a low priority issue which would still need a lot of preparation for a balanced implementation. How would a balanced way to obtain the required materials work? How could we make it so Inscription is not severely nerfed when you can buy the mats from a vendor? What would happen to the glyphs and materials once WOTLK drops? This requires a lot of time and manpower we currently need for other issues.

While it is true that other classes already have their BiS glyphs available, this can be seen more as an oversight than a wanted feature.

It basically boils down to: We don't think its a necessarily a bad idea to release the glyphs early. We just don't think its currently worth putting in all the effort it would need in order to implement this properly while we are packed with more time sensitive issues.

**Reply by Hyperbearr** [reply]
Going to be blunt and concise rather than risk rambling, apologies if that comes off as rude. 

- What class balance? PVP and PVE are already dominated by Hunters and Warlocks, Holy Paladins already significantly outperform all other healers in both PVP and PVE etc etc. Class balance is never going to exist on a PTE server and it definitely doesn't exist here and that is fine, we all know what we signed up for. 

- It takes nothing away from later brackets, glyphs are consumable under the WOTLK implementation so we will have to keep farming them from now until ICC and you can remove the vendor as each bracket is released. At the same time you make the glyphs exorbitantly expensive (100g each? 200g each?) which means that stockpiling them for later sale will be less economic than simply making them with Inscription. This also has an economic perk in that it removes gold from the economy, which is rapdily becoming an issue as materials continue to inflate and both players and guilds collect obscene amounts of gold. We all know of handfuls of players with 10's or even 100's of thousands of gold, so adding significant gold sinks would be benificial at the current juncture. 

- Again, it doesn't need to be balanced and it doesn't need to be materials, make it gold and call it a day. At which point its less work than many of the other recent introductions like the Tanaris vendor, custom costumes or transmog, all of which I assume are low priority issues (compared to the servers mission statement) but which have still managed to obtain the requisite development time.

**Reply by Hacki95** [reply]
This is more of a personal view now and might not reflect the staffs view, but here we go.

- It is true that class balance is already pretty onesided. And thats why we are extremely cautious whenever we touch anything which will change this further.

- It takes content away in that sense that people with inscription can already pre-farm things for later brackets. Even with high prices I don't think this will hold people back from making some smart business moves we don't even think about right now. And thats also part of the problem. Whenever we consider releasing things earlier or out of order we have to take all kinds of factors into perspective - which again is a massive amount of work.

- Be aware that those "fun" things like the transmog, customes, events etc. are done by staff who felt like doing it, while most of the bug fixes etc are done by completely different volunteers and devs. We have things we need to do and we have things we want to do and those two things are pretty much seperate.

**Reply by plasma-93** [reply]
To counterpoint, after chatting with some guildies and others about it:

- PvP balance is hurt a lot more from not having the glyphs (Stoneclaw Glyph for example, some classes don't even have good pvp glyphs to use at all) and the PvP on the server is already a mess atm.

- I'm not sure if I even understand the 'more content later' part.. certain builds will just be better regardless of glyphs, and some builds ONLY become playable with glyphs. It's either 'more content forever', or 'less content for a while'. You don't eek more juiice out of the players by limiting the intended glyph availability here. You just have people that for example, would play Shadow Priest, just not playing their preferred class and in some situations that just means they won't play. This also has shaped the class makeup fo the server, where the classes that have access to some of their better glyphs perform better, and play better,  and makeup a larger portion of the server. (MM Hunters, Arms Warriors)

- Your player base disagrees that this is a 'low priority issue' based on the amount of people asking for it that want more specs to be viable, increasing player agency and replay-abilit. I'm an officer in Synergy, one of, if not the largest guild on the server, and often people have expressed it's not an 'if', it's a 'when' for the glyphs - believing that it is 'obviously' good for the server.

- It would be a good way to implement a gold sink, 100g(tbc)/200g(wrath) for example. Simply changing the progression module so _if tbc, remove 301-375 glyphs from vendor_, and _if wotlk remove 376-450 from vendor_ and during the downtime from vanilla>tbc and tbc>wrath remove all the item id's for the specific glyphs from everyones inventory and talents, and re-earned through trading some gold to a scribe :) 

-Finally, in regards to the amount of effort required.. I struggle to see how this is the case. One custom vendor with gold costs is all that's needed to make a bunch of people happy. Let the glyph-less builds thrive! If someone made the module, would it be considered? I don't mean to diminish anyone's work, but I struggle to see more than a afternoons work for this sort of addition.

**Reply by Arbit0** [reply]
I don't know enough about glyphs outside of the ones I myself use(I play MM hunter(s)). However I'd like the server even more if there was more of an even playing field when it comes to classes, as of right now the DPS charts for example is HEAVILY dominated by hunters/warriors. Shake things up a bit!

**Reply by Hacki95** [reply]
> -Finally, in regards to the amount of effort required.. I struggle to see how this is the case. One custom vendor with gold costs is all that's needed to make a bunch of people happy. Let the glyph-less builds thrive! If someone made the module, would it be considered? I don't mean to diminish anyone's work, but I struggle to see more than a afternoons work for this sort of addition.

Since I said its mainly about the work load, sure if somebody here wants to do it for us - we wont hold anyone back and we would gladly use it.

**Reply by Annamaria-CC** [reply]
> > -Finally, in regards to the amount of effort required.. I struggle to see how this is the case. One custom vendor with gold costs is all that's needed to make a bunch of people happy. Let the glyph-less builds thrive! If someone made the module, would it be considered? I don't mean to diminish anyone's work, but I struggle to see more than a afternoons work for this sort of addition.
> 
> Since I said its mainly about the work load, sure if somebody here wants to do it for us - we wont hold anyone back and we would gladly use it.

I like the idea, so i'll do it.

**Reply by Si1ker** [reply]
> 

@temperrr I have the majority already done btw, you may want to keep an eye on zul'gurub stuff since is soon to release.

**Comment by oterfesk** [comment]
If this goes through, will all glyphs be added? Or only glyphs for which we have abilites? Since this change goes more so at the core of the game, adding every glyph will inevitably lead to more discussions of such changes like "well.. since you did the glyphs why not add the rest of the abilites aswell?" Surely the people wont demand an arm after you give them a finger.. right?

**Comment by Nyeriah** [comment]
Hey,

Following up Hacki's response to this, I would like to mention that we're reevaluating this suggestion and what we could possibly do to address the unacessible glyphs temporarily until they're available, later in the progression.

We appreciate the willingness to help and all the support we've received from our community, which majorly seems to want to see this happening.

However, while all the help we receive is certaily welcome, there's no need independently work on this as we're already working on *something* ourselves. Therefore, it would be counterproductive to have several different persons coming forth each with a different solution, that not necessarily aligns with our goals, so I'd like to, once again, thank those who tried to implement this but no further community support is required.

Stay tuned in for further updates about this.

Regards.

**Comment by Si1ker** [comment]
4 hours wasted, nice 🤡 gl
